src/main/java/com/uber/jenkins/phabricator/PhabricatorBuildWrapper.java

```diff
@@ -114,6 +119,10 @@ public Environment setUp(AbstractBuild build,
             if (!createCommit) {
                 patchCommand.add("--nocommit");
             }
+            // Use a globally-configured conduit token, if applicable
+            if (!CommonUtils.isBlank(conduitToken)) {
+                patchCommand.add("--conduit-token=" + this.getConduitToken());
```

Comment:
The head commit on this branch now suppresses the command name, which alleviates any security concerns but makes debugging harder. Not sure how I feel about it.